apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```
@@ -1,22 +1,26 @@
-import style from '../../../style/code-studio/curriculum_catalog_container.module.scss';
-import HeaderBanner from '../HeaderBanner';
-import CourseCatalogBannerBackground from '../../../static/curriculum_catalog/course-catalog-banner-bg.png';
-import CourseCatalogIllustration01 from '../../../static/curriculum_catalog/course-catalog-illustration-01.png';
-import CourseCatalogNoSearchResultPenguin from '../../../static/curriculum_catalog/course-catalog-no-search-result-penguin.png';
+import React, {useState, useEffect} from 'react';
+
 import {Heading5, BodyTwoText} from '@cdo/apps/componentLibrary/typography';
-import CurriculumCatalogFilters from './CurriculumCatalogFilters';
-import CurriculumCatalogCard from '@cdo/apps/templates/curriculumCatalog/CurriculumCatalogCard';
-import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
 import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import CurriculumCatalogCard from '@cdo/apps/templates/curriculumCatalog/CurriculumCatalogCard';
 import {
   getSimilarRecommendations,
   getStretchRecommendations,
 } from '@cdo/apps/util/curriculumRecommender/curriculumRecommender';
 import {tryGetSessionStorage, trySetSessionStorage} from '@cdo/apps/utils';
+import i18n from '@cdo/locale';
+
+import CourseCatalogBannerBackground from '../../../static/curriculum_catalog/course-catalog-banner-bg.png';
+import CourseCatalogIllustration01 from '../../../static/curriculum_catalog/course-catalog-illustration-01.png';
+import CourseCatalogNoSearchResultPenguin from '../../../static/curriculum_catalog/course-catalog-no-search-result-penguin.png';
+import HeaderBanner from '../HeaderBanner';
+
+import {curriculumDataShape} from './curriculumCatalogConstants';
+import CurriculumCatalogFilters from './CurriculumCatalogFilters';
+
+import style from '../../../style/code-studio/curriculum_catalog_container.module.scss';
```

If anything in this change was indeed responsible for those eyes failures last time rather than just flakiness, I'd guess it's one of these `import style` lines